connectors/gitlab.mdx

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
https://github.com/danswer-ai/danswer-docs/blob/main/mint.json#L74

You need to add it to the list or it won't show